storage/store_test.go

```diff
@@ -385,12 +385,6 @@ func TestStoreVerifyKeys(t *testing.T) {
 	if err := store.ExecuteCmd(proto.Put, pArgs, pReply); err != nil {
 		t.Fatalf("unexpected error on put to meta2 value: %s", err)
 	}
-	// Try a put to txn record for a meta2 key.
```

Comment:
Good that you pointed this out. I don't know what I was thinking and restored this test and actually fixed it because now we're encoding the transaction key, which lengthens it considerably. I'm also now testing for the appended txn ID. Those keys can now get pretty damn large in the txn-local-prefix + encoded(meta2 + KeyMax) + UUID case!